snomed/com.b2international.snowowl.snomed.ecl/src/com/b2international/snowowl/snomed/ecl/Ecl.java

```diff
@@ -27,6 +27,7 @@
 	public static final String ANY = "*";
 	public static final int MAX_CARDINALITY = -1;
 	public static final Joiner OR_JOINER = Joiner.on(" OR ");
+	public static final Joiner AND_JOINER = Joiner.on(",");
```

Comment:
Could this introduce confusion when an expression is deserialized for representation on the UI? I mean for disjunction we use 'OR' where there is no alternative, but for conjunction we use ',' where there is a written alternative 'AND'? (this does not take the credit for the PR itself, just being :philosoraptor:)

Reply:
Hmm, not sure, I assume if someone is writing ECL, he knows the actual ECL syntax, and commas are used in conjunctions. To simplify and make it consistent, I can change it to use the actual `AND` word as separator, instead of the `,`.